sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskBulkEditData.java

```diff
@@ -0,0 +1,44 @@
+package de.symeda.sormas.ui.task;
+
+import de.symeda.sormas.api.EntityDto;
+import de.symeda.sormas.api.task.TaskPriority;
+import de.symeda.sormas.api.task.TaskStatus;
+import de.symeda.sormas.api.user.UserReferenceDto;
+
+public class TaskBulkEditData extends EntityDto {
+
+	private static final long serialVersionUID = -7234609753914205675L;
+
+	public static final String TASK_REASSIGNED_TO = "taskReassignedTo";
+	public static final String TASK_PRIORITY = "taskPriority";
+	public static final String TASK_STATUS = "taskStatus";
+
+
+	private UserReferenceDto taskReassignedTo;
```

Comment:
Change this to `assignee` and update all similarly named fields and properties (caption should be "Change task assignee")